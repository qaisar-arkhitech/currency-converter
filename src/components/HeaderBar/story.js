import React from "react"
import HeaderBar from "."

export default {
  title: "HeaderBar",
  storyFn() {
    return (
      <div>
        <HeaderBar />
      </div>
    )
  },
}
