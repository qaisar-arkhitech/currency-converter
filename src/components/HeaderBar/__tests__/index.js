import React from "react"
import renderer from "react-test-renderer"
import HeaderBar from ".."

const createProps = () => ({
  classes: {},
})

describe("<HeaderBar />", () => {
  it("Does not explode", () => {
    const props = createProps()
    const component = renderer.create(<HeaderBar {...props} />).toJSON()
    expect(component).toMatchSnapshot()
  })
})
