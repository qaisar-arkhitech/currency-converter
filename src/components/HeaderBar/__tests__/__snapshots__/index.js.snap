// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderBar /> Does not explode 1`] = `
<View>
  <Text>
    HeaderBar
  </Text>
</View>
`;
